### DISCUSSION TITLE
Unescaped sample broken right now?

### DISCUSSION THREAD

**Discussion by socmag** [taocpp/PEGTL#56] [Q&A] [answered]
Hello, I'm just trying out PEGTL - seems totally awesome.

I can't seem to get the unescaped sample to work at the moment
Get this error no matter what I send it as an argument:

`terminate called after throwing an instance of 'tao::pegtl::parse_error'
  what():  argv[1]:1:0(0): parse error matching tao::pegtl::pad<example::literal, tao::pegtl::ascii::blank, tao::pegtl::ascii::blank>
`

I literally just started looking at PEGTL today, so I wouldn't be at all surprised if this is my screw up.

This is on the MASTER branch synced today btw.

Should I go back on to the 1.3.1 release for now?

thanks!

**Comment by socmag** [comment]
Just tried it with the 1.3.1 release and get the same thing.

Maybe I'm confused what I'm supposed to be passing on the command line.

I've tried:

```
./unescape abc
./unescape "abc"

```
and a bunch of other things... no luck (gcc 6.2.1-7 btw)

**Comment by d-frey** [comment, marked as answer]
You need to pass in a string (with escaped characters if you want) including the surrounding double-quotes. In order to do so, it is not sufficient to simply pass double-quotes on the command line, as the shell will interpret them and won't pass it to the example code. The correct way to pass them is:

    ./unescape '"abc"'

This should work in both the 1.3.1 as well as the current master branch.

**Comment by socmag** [comment]
Aha, that was the one combo I didn't try,
Works like a champ, thanks!
